birdhouse/pavics-compose.sh

```diff
@@ -25,8 +25,6 @@ VARS='
   $POSTGRES_PAVICS_PASSWORD
   $POSTGRES_MAGPIE_USERNAME
   $POSTGRES_MAGPIE_PASSWORD
-  $ALERTMANAGER_ADMIN_EMAIL_RECEIVER
-  $SMTP_SERVER
```

Comment:
+1 on this one
I was wondering why these variable needed to be defined even if monitoring wasn't enabled.
This offers more flexibility, great.